CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog of postpic
 
 ## current master
-
-
+2017-10-10
```

Comment:
can you at first add a section `incompatible changes` please? This is the most important part and also needs to be described well enough for users to understand what adjustments they have to make to their code.